collection-input/cesm1-le-collection.yml

```diff
@@ -237,6 +246,47 @@ data_sources:
             member_id: 105
             has_ocean_bgc: True
             ctrl_branch_year: 402
+          - case: b.e11.B20TRC5CNBDRD.f09_g16.OIC.001d
```

Comment:
I would prefer to omit the OIC runs as memebers of the 20C ensemble: they are different enough that they should not be included automatically. 

I would suggest adding a new experiment: 20C_OIC

Reply:
> I would suggest adding a new experiment: 20C_OIC

Done!